extendedset/src/main/java/io/druid/extendedset/intset/ConciseSet.java

```diff
@@ -75,7 +73,7 @@
    * User for <i>fail-fast</i> iterator. It counts the number of operations
    * that <i>do</i> modify {@link #words}
    */
-  protected transient volatile int modCount = 0;
+  protected transient int modCount = 0;

```

Comment:
Looks like this is never being read, even in subclasses. We should just get rid of it if it's not doing anything useful.